rootfs/scheduler/__init__.py

```diff
@@ -568,42 +583,53 @@ def run(self, namespace, name, image, entrypoint, command, **kwargs):
         if unhealthy(response.status_code):
             error(response, 'create Pod in Namespace "{}"', namespace)
 
-        data = ''
-        duration = 30
-        iteration = 1
-        while (iteration < duration):
-            try:
+        labels = {
+            'app': namespace,
+            'type': 'run',
+            'version': kwargs.get('version'),
```

Comment:
Sorry, I'm referring to the label instead of the keyword argument. `POD_TEMPLATE` [uses `$appversion`](https://github.com/helgi/controller/blob/7e44adb167eff9504774bb8283473ebadd66fd88/rootfs/scheduler/__init__.py#L93) for the application version number. `$version` is used for the API version number.